improve text-to-image task page (#889)
huggingface/huggingface.js: packages/tasks/src/tasks/text-to-image/about.md

Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.

> ##Task Variants

Reviewer: ```suggestion
## Task Variants
```

### Image Editing

Image editing with text-to-image models involves using text prompts to describe the wanted changes in an image and then follow them.

> - Synthetic image editing: using text-to-image models to make adjustments to images that were initially created using an input prompt, while preserving the overall meaning or context of the original image.

Reviewer: ```suggestion
- **Synthetic image editing:** Adjusting images that were initially created using an input prompt while preserving the overall meaning or context of the original image.
```

  ![Examples](https://datasets-server.huggingface.co/assets/diffusers/diffusers-images-docs/--/b20ecaa3f61372174c854e09fc856fdcce6f8494/--/default/train/0/image/image.png?Expires=1725455983&Signature=ykj3EnAENI6goXc7qI2Toq~P8P5IdS1DqNbSfH8vhgrdwaJoGH2cUbXWRgVAndhrHvRjrTTcU3YOyoExnot7zEhauyUEcqr-evRHDmGgfar52uEmfLbLCtNAcRK9Q85QOifupIH-X9x3rBUM03B0RIkHuto6wwRBAHireqr7QcD8hYRaNzACXrTbt-U7wHosZS8R1pdc3FDt7fDc3Qwh8XL0YoJqAoK8X8JnZEXIWTfGnCpygPBDbseDlYEzegGKzClAUgigQbomUk733VNtB3ol396uYkHCcjqjtgdhtEfAWQz-xM4eAhHpI~YEn7RQqRjB0RD0bPd1nHRU0wGUqA__&Key-Pair-Id=K3EI6M078Z3AC3)

> - Real image editing: similar to synthetic image editing, except we're using real photos/images. This task is usually more complex, as it involves first obtaining a latent representation of the image, in the latent domain of the model that it can then manipulate.

Reviewer: ```suggestion
- **Real image editing**: This is very similar to synthetic image editing, real photos/images are used. This task is usually more complex.
```
I think we don't need to give much technical details since this page is beginner (if you'd like, you can explain in even simpler terms)

  ![Examples](https://datasets-server.huggingface.co/assets/diffusers/diffusers-images-docs/--/default/train/1/image/image.jpg?Expires=1725453082&Signature=MOCeELTChydgLRZT9ws8owCraSVrdcm6c7Vlnsi23rJ1Ocigl6gjRtXwmjVDCKuG2fB6Hw0Tmn8ZR0M7FPiA2fXpSuPEW4iJMoeQNiNCtkSSjjDisDXbBSRXW1TXJ-Z2c~VoJ4lmmeUdFpyFZ9W~BlI6r2xQLltfU400XKPe~UgE-vJ~xr9ni8zZmyYt1kVtV9Et~EBzWCQkKc2DO9gI9HnEg9z2hxDHp8Bak0HBRARM4ObhRYxieWqO4hOg1HVk4LSt2E8emIuDmhPUU4v8L097yFcI4D6JeoyNNn0q6nKQZqAZIzwP8iiLqqhSv~mJsO7YGnQck1-bzA~gAiVMpg__&Key-Pair-Id=K3EI6M078Z3AC3)

### Personalization

> Personalization refers to techniques used to customize text-to-image models, where we introduce new subjects/concepts to the model so that we can then use the model to generate new images of those subjects with a text prompt. For example, one can use these techniques to generate images of themselves, using as little as one reference image. These include teaching the model a new concept both in training free manner or through fine-tuning.

Reviewer: ```suggestion
Personalization refers to techniques used to customize text-to-image models. In this technique, we introduce new subjects/concepts to the model and then use the model to generate new images of those subjects with a text prompt. 

For example, one can use these techniques to generate images of themselves, using as little as one reference image. These include teaching the model a new concept both in a training-free manner or through fine-tuning.
```

I am actually irritated by personal DreamBooths, to be honest, especially with the recent Korean cases where it's weaponized against women. Can you change the example?

Author: That's a good point. But I also feel like it's such a major use case, it's a bit odd not to mention it. Cause mentioning style Loras can also potentially be problematic on the artists angle

Author: perhaps mentioning both?

Reviewer 2: These are good points. A couple of alternatives:
- Use "your dog" as an example (I did this in my suggestion). I agree with Linoy that this somewhat sweeps a major use case under the rug.
- Be open about it and include a comment about using these features responsibly, respectfully, and ethically.

Architects can utilise the models to construct an environment based out on the requirements of the floor plan. This can also include the furniture that has to be placed in that environment.

## Task Variants

### Image Editing

> Image editing with text-to-image models involves using text prompts to describe the wanted changes in an image and then follow them.

Reviewer 2: ```suggestion
Some text-to-image models can carry out modifications of an input image following the instructions provided in a text prompt.
```

Author: "some...can carry" - I feel like this part might be a bit misleading - as it's not an inherent capability of only some text to image models, wdyt?

Reviewer 2: Sure, feel free to adjust!

- **Synthetic image editing**: Adjusting images that were initially created using an input prompt while preserving the overall meaning or context of the original image.

>   ![Examples](https://datasets-server.huggingface.co/assets/diffusers/diffusers-images-docs/--/b20ecaa3f61372174c854e09fc856fdcce6f8494/--/default/train/0/image/image.png?Expires=1725455983&Signature=ykj3EnAENI6goXc7qI2Toq~P8P5IdS1DqNbSfH8vhgrdwaJoGH2cUbXWRgVAndhrHvRjrTTcU3YOyoExnot7zEhauyUEcqr-evRHDmGgfar52uEmfLbLCtNAcRK9Q85QOifupIH-X9x3rBUM03B0RIkHuto6wwRBAHireqr7QcD8hYRaNzACXrTbt-U7wHosZS8R1pdc3FDt7fDc3Qwh8XL0YoJqAoK8X8JnZEXIWTfGnCpygPBDbseDlYEzegGKzClAUgigQbomUk733VNtB3ol396uYkHCcjqjtgdhtEfAWQz-xM4eAhHpI~YEn7RQqRjB0RD0bPd1nHRU0wGUqA__&Key-Pair-Id=K3EI6M078Z3AC3)

Reviewer 2: These URLs don't work for me and seem to contain authentication info. Are you meaning to use files from https://huggingface.co/datasets/diffusers/diffusers-images-docs/tree/main?

Author: shoot, yeah thats my bad!

- **Real image editing**: similar to synthetic image editing, except we're using real photos/images. This task is usually more complex.

>   ![Examples](https://datasets-server.huggingface.co/assets/diffusers/diffusers-images-docs/--/default/train/1/image/image.jpg?Expires=1725453082&Signature=MOCeELTChydgLRZT9ws8owCraSVrdcm6c7Vlnsi23rJ1Ocigl6gjRtXwmjVDCKuG2fB6Hw0Tmn8ZR0M7FPiA2fXpSuPEW4iJMoeQNiNCtkSSjjDisDXbBSRXW1TXJ-Z2c~VoJ4lmmeUdFpyFZ9W~BlI6r2xQLltfU400XKPe~UgE-vJ~xr9ni8zZmyYt1kVtV9Et~EBzWCQkKc2DO9gI9HnEg9z2hxDHp8Bak0HBRARM4ObhRYxieWqO4hOg1HVk4LSt2E8emIuDmhPUU4v8L097yFcI4D6JeoyNNn0q6nKQZqAZIzwP8iiLqqhSv~mJsO7YGnQck1-bzA~gAiVMpg__&Key-Pair-Id=K3EI6M078Z3AC3)

Reviewer 2: Same

### Personalization

> Personalization refers to techniques used to customize text-to-image models, where we introduce new subjects/concepts to the model so that we can then use the model to generate new images of those subjects with a text prompt.

Reviewer 2: ```suggestion
Personalization refers to techniques used to customize text-to-image models. We introduce new subjects or concepts to the model, which the model can generate when we refer to them with a text prompt.
```

> For example, one can use these techniques to generate images of themselves, using as little as one reference image. These include teaching the model a new concept both in training free manner or through fine-tuning.

Reviewer 2: ```suggestion
For example, you can use these techniques to generate images of your dog in imaginary settings, after you have taught the model a few reference images of the subject (or just one in some cases). Teaching the model a new concept can be achieved through fine-tuning, or by using training-free techniques.
```
